<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ConfigureDialog</class>
 <widget class="QDialog" name="ConfigureDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>801</width>
    <height>543</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QListWidget" name="tab_list">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="maximumSize">
        <size>
         <width>150</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="horizontalScrollBarPolicy">
        <enum>Qt::ScrollBarAlwaysOff</enum>
       </property>
       <property name="autoScrollMargin">
        <number>16</number>
       </property>
       <property name="editTriggers">
        <set>QAbstractItemView::NoEditTriggers</set>
       </property>
       <property name="iconSize">
        <size>
         <width>50</width>
         <height>50</height>
        </size>
       </property>
       <property name="viewMode">
        <enum>QListView::ListMode</enum>
       </property>
       <item>
        <property name="text">
         <string>General</string>
        </property>
        <property name="textAlignment">
         <set>AlignCenter</set>
        </property>
        <property name="icon">
         <iconset theme="general_icon">
          <normaloff>.</normaloff>.</iconset>
        </property>
        <property name="flags">
         <set>ItemIsSelectable|ItemIsEnabled</set>
        </property>
       </item>
       <item>
        <property name="text">
         <string>System</string>
        </property>
        <property name="textAlignment">
         <set>AlignCenter</set>
        </property>
        <property name="icon">
         <iconset theme="system_icon">
          <normaloff>.</normaloff>.</iconset>
        </property>
        <property name="flags">
         <set>ItemIsSelectable|ItemIsEnabled</set>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Enhancements</string>
        </property>
        <property name="textAlignment">
         <set>AlignCenter</set>
        </property>
        <property name="icon">
         <iconset theme="enhancements_icon">
          <normaloff>.</normaloff>.</iconset>
        </property>
        <property name="flags">
         <set>ItemIsSelectable|ItemIsEnabled</set>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Controller</string>
        </property>
        <property name="textAlignment">
         <set>AlignCenter</set>
        </property>
        <property name="icon">
         <iconset theme="input_icon">
          <normaloff>.</normaloff>.</iconset>
        </property>
        <property name="flags">
         <set>ItemIsSelectable|ItemIsEnabled</set>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <property name="spacing">
        <number>0</number>
       </property>
       <item>
        <widget class="QTabWidget" name="general_tab_group">
         <property name="currentIndex">
          <number>0</number>
         </property>
         <widget class="ConfigureGeneral" name="general_tab">
          <attribute name="title">
           <string>General</string>
          </attribute>
         </widget>
         <widget class="ConfigureWeb" name="web_tab">
          <attribute name="title">
           <string>Web</string>
          </attribute>
         </widget>
         <widget class="ConfigureDebug" name="debug_tab">
          <attribute name="title">
           <string>Debug</string>
          </attribute>
         </widget>
        </widget>
       </item>
       <item>
        <widget class="QTabWidget" name="system_tab_group">
         <property name="currentIndex">
          <number>0</number>
         </property>
         <widget class="ConfigureSystem" name="system_tab">
          <attribute name="title">
           <string>System</string>
          </attribute>
         </widget>
         <widget class="ConfigureAudio" name="audio_tab">
          <attribute name="title">
           <string>Audio</string>
          </attribute>
         </widget>
        </widget>
       </item>
       <item>
        <widget class="QTabWidget" name="enhancements_tab_group">
         <widget class="ConfigureGraphics" name="graphics_tab">
          <attribute name="title">
           <string>Graphics</string>
          </attribute>
         </widget>
        </widget>
       </item>
       <item>
        <widget class="QTabWidget" name="controller_tab_group">
         <property name="currentIndex">
          <number>0</number>
         </property>
         <widget class="ConfigureInput" name="input_tab">
          <attribute name="title">
           <string>Input</string>
          </attribute>
         </widget>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QDialogButtonBox" name="buttonBox">
       <property name="standardButtons">
        <set>QDialogButtonBox::Apply|QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ConfigureGeneral</class>
   <extends>QWidget</extends>
   <header>configuration/configure_general.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ConfigureSystem</class>
   <extends>QWidget</extends>
   <header>configuration/configure_system.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ConfigureAudio</class>
   <extends>QWidget</extends>
   <header>configuration/configure_audio.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ConfigureDebug</class>
   <extends>QWidget</extends>
   <header>configuration/configure_debug.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ConfigureInput</class>
   <extends>QWidget</extends>
   <header>configuration/configure_input.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ConfigureGraphics</class>
   <extends>QWidget</extends>
   <header>configuration/configure_graphics.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ConfigureWeb</class>
   <extends>QWidget</extends>
   <header>configuration/configure_web.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>ConfigureDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>400</x>
     <y>521</y>
    </hint>
    <hint type="destinationlabel">
     <x>400</x>
     <y>271</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>ConfigureDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>400</x>
     <y>521</y>
    </hint>
    <hint type="destinationlabel">
     <x>400</x>
     <y>271</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
